docs/design/README.md

```diff
@@ -0,0 +1,30 @@
+# Design Docs
+Generally, design docs are on Google docs:
+
+## Conventions
+* [Versioning of Constraint Templates](https://docs.google.com/document/d/1vB_2wm60WCVLXoegMrupqwqKAuW6gbwEIxg3vBQj6cs/edit)
+
+## In Development
+* [Metrics Design Issue](https://github.com/open-policy-agent/gatekeeper/issues/157#issuecomment-553015292)
+
+## Proposed
+* [Logging Design Doc](https://docs.google.com/document/d/1ap7AKOupNcR_42s8mkSh5FV9eteXTd4VCqelKst73VY/edit)
+* [Namespace-Scoped Constraints](https://docs.google.com/document/d/1-pY7B5C6R0fjUbDu8izlcP7MSUDVDHv5XK16BFyTGRc/edit#heading=h.w8j68o8vjdts)
+* [Mutation Revised Design Doc](https://docs.google.com/document/d/1G7WgZKx1Y3VOTUjrqn7DjDaZgSKCIZowILm_I6psrw0/edit#heading=h.mtvdjag5uj9)
+   * [Mutation Initial Design Doc](https://docs.google.com/document/d/1qTHwqoUX8AL2jodyWKB_2szrGDwhi14Ra_LlQ-ogtck/edit#heading=h.iu1ppjy7g7j)
+
+## Implemented
+* [V3 Accepted Design](https://docs.google.com/document/d/1yC4wgpVoJj6ngYnSTtO-HeaIBl05gla562sD7qKPy3M/edit#heading=h.z0bjqzl81dpe)
```

Comment:
This one resolves to `Gatekeeper Policy Library Design Proposal` is that right?

Reply:
Ya, that's what the doc was titled, even if it's a bit of a misnomer.

It was the group doc we all worked off of after the original proposals.

Reply:
ok. just double checking